docs/changelog/83637.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 83637
+summary: "Fix: ip prefix bucket reduction"
+area: Aggregations
+type: bug
+issues: []
```

Comment:
Do I need to add 83321 here?

Reply:
I have no idea. I don't really understand this process totally.